packages/components/src/index.js

```diff
@@ -1,10 +1,7 @@
 // @flow
-export { default as UIAmountInput } from './UIAmountInput';
 export * from './UIActionComponent';
 export { default as UIComponent } from './UIComponent';
 export { default as UIDetailsTable } from './UIDetailsTable';
 export { default as UIDetailsView } from './UIDetailsView';
-export * from './UIDetailsInput';
 export { default as UIPureComponent } from './UIPureComponent';
 export { default as UIScaleButton } from './UIScaleButton';
```

Comment:
Can you make a new component for that in UIKit, instead of this one, please?
It's a small one, but would let us kill `UIActionComponent` and `UIPureComponent` at once :)

Reply:
I'm actually curious what it's used for?

Reply:
Right now this kind of button is used for PromoCards in Surf

Reply:
oh, and also for transactions bubbles in the chat ;)

Reply:
aha, so it's not an actual button (with it's own design), it's kinda like `TouchableScale`

Reply:
True!